plugins/usbdmx/README.md

```diff
@@ -7,7 +7,8 @@ This plugin supports various USB DMX devices including:
 * AVLdiy D512
 * DMXControl Projects e.V. Nodle U1
 * DMXCreator 512 Basic
-* Eurolite
+* Eurolite USB-DMX512 PRO
+* Eurolite USB-DMX512 PRO MK2
```

Comment:
It might be worth explicitly mentioning enable_eurolite_mk2 here?

Reply:
Sure